client/src/app/site/pages/meetings/pages/participants/pages/participant-list/services/participant-list-sort.service/participant-list-sort.service.ts

```diff
@@ -20,8 +20,8 @@ export class ParticipantListSortService extends BaseSortListService<ViewUser> {
      */
     private userSortOptions: OsSortingOption<ViewUser>[] = [
         { property: `full_name`, label: `Full name` },
-        { property: `first_name`, label: `Given name` },
-        { property: `last_name`, label: `Surname` },
+        { property: [`first_name`, `last_name`], label: `Given name` },
+        { property: [`last_name`, `last_name`], label: `Surname` },
```

Comment:
```suggestion
        { property: [`last_name`, `first_name`], label: `Surname` },
```
probably you meant this?